Library/Homebrew/cmd/install.rb

```diff
@@ -143,7 +145,12 @@ def check_cellar
   def perform_preinstall_checks
     check_ppc
     check_writable_install_location
-    check_xcode
+    if MacOS::Xcode.installed?
+      check_xcode
+    else
+      opoo "You have not installed Xcode."
+      puts "Bottles may install correctly, but builds will fail!"
```

Comment:
It's potentially only worth printing this warning about builds potentially failing *if* a user is building from source. Unless there's something to warn them about, I don't think we need to run a warning on `install` every time.